docs/meaning_for_configurations_zh.md

```diff
@@ -1,3 +1,245 @@
+        name='base_group',
+        # How parameters in each client aggregate. Default to be "avg", which means a simple average.
+        aggregation_method='avg',
+        # What parameters in each client should be aggregated.
+        aggregation_parameters=dict(
+            # For default case, every parameter should be aggregated.
+            name='all'
+        ),
+    ),
+    # Launcher configurations.
+    launcher=dict(
+        # For the simplest launcher, serial is the suitable choice.
+        name='serial'
+        # If you want to use multiprocess to accelerate the training process, you can use the following setting.
+        # name='multiprocessing',
+        # num_proc=2
+        # ``num_proc`` refers to the number of processes used in your program.
+        # For the default setting, if your os is linux, the multiprocessing mode is enabled.
+        # You can overwrite the default settings by yourself.
+    ) if platform.system().lower() != 'linux' else dict(name='multiprocessing', num_proc=2),
+    # Other configurations.
+    other=dict(
+        # Frequency for testing. For example, `test_freq=3` means the performance is tested every 3 global epochs.
+        test_freq=3,
+        # What is the logging directory of this experiment.
+        # If the directory does not exist, it will be created automatically.
+        # If the directory already exists, some parts might be over-written, which should be carefully inspected.
+        logging_path='./logging/default_experiment',
+        # The saved model checkpoint to start from. If it is set to ``None``, the training process
+        # will start from scratch.
+        resume_path=None,
+        # Whether to print config is the command line.
+        print_config=False,
+    ),
+)
+```
+
+## 其他教程
+
+### learn.scheduler
+
+此键用于控制学习率调度器。除了 "fix" 外，`learn.scheduler` 键还具有多种类型。例如，我们可以使用余弦函数定义一个调度器：
+
+```python
+scheduler=dict(
+    # Cosine learning rate scheduler.
+    name='cos'
+    # The `min_lr` is the lower bound of learning rate.
+    min_lr=1e-6,
+    # This means that the learning rate will decay progressively for 30 global epochs and finally reaches `min_lr`.
+    decay_round=30
+)
+```
+
+对于其他类型的调度器，请参考 `fling.utils.LRScheduler`。
+
+### learn.finetune_parameters
+
+这个键用于控制哪些参数应该进行微调。除了 "all" 之外，`learn.finetune_parameters` 键还具有其他值。例如，如果您只想微调参数名称中**包含**关键词 "fc" 或 "bn" 的参数，可以这样写：
+
+```python
+finetune_parameters=dict(
+    # This means to only fine-tune parameters whose names have keywords listed in `keywords`
+    name='contain',
+    keywords=['fc', 'bn']
+)
+```
+
+相反，如果您想微调参数名称**不包含**关键字 "fc" 或 "bn" 的参数，可以这样写：
+
+```python
+finetune_parameters=dict(
+    # This means to only fine-tune parameters except those whose names have keywords listed in `keywords`
+    name='except',
+    keywords=['fc', 'bn']
+)
+```
+
+`group.aggregation_parameters` 控制在联邦学习框架下应该聚合哪些参数，对于它的设置和上面所述的 `learn.finetune_parameters` 是类似的。
```

Comment:
Please an example here. See FedPer config: https://github.com/kxzxvbk/Fling/blob/main/argzoo/cifar10/cifar10_fedper_resnet_config.py.
Explain concretely what part is personalized, what part is fine-tuned.